.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,20 +13,20 @@
 -->
 
 This PR contains:
-
 - [ ] bugfix
 - [ ] feature
-- [ ] refactor
-- [ ] tests
+- [ ] refactoring
 - [ ] documentation
-- [ ] metadata
-
-### Breaking Changes?
+- [ ] other
 
+Are tests included?
 - [ ] yes
+
+Breaking Changes?
+- [ ] yes (please describe clearly below)
```

Comment:
since we're removing the heading, let's make `(please describe clearly below)` stand out some more by bolding it. emphasis here is a good thing.